### PR TITLE
Fix 2.8.0 Release

### PR DESCRIPTION
**Pull request by bkb2135**
Fix various bugs in 2.8.0 Release

### CODE REVIEW

**Review by Hollyqui — APPROVED** [review]
LGTM

**Review by dbobrenko — APPROVED** [review]
LGTM

File: prompting/tasks/programming_task.py [hunk 1]
```diff
@@ -30,21 +31,26 @@ class ProgrammingRewardConfig(BaseRewardConfig):
     Original code:
     {file_content}
 
-    Respond only with the new and modified code!
+    Respond only with the new and modified code! Make sure to respond the with the entire code, not just a part of it.
```
Comment (dbobrenko) [inline prompting/tasks/programming_task.py:34]:
```suggestion
    Respond only with the new and modified code! Make sure to respond with the entire code, not just a part of it.
```

File: notebooks/testing.ipynb [hunk 2]
```diff
@@ -1,5 +1,161 @@
 {
  "cells": [
+  {
```
Comment (dbobrenko) [inline notebooks/testing.ipynb:3]: We should remove ipynbs, otherwise it will keep growing the project size

**Review by dbobrenko — COMMENTED** [review]
Additional comments

**Pull request merged**